runtime/shared_common/include/SCQueryFunctions.h

```diff
@@ -33,6 +33,23 @@ extern "C"
 {
 #endif
 
+ #if defined(__GNUC__)
+ /*
+  * Source files which include (directly or indirectly) this file
+  * but do not use the functions defined below may fail to compile unless the "unused" attribute
+  * is applied.
+  * Note that Microsoft compilers do not allow this attribute.
+  */
+#if !defined(J9VM_SHRTEST)
+#if !defined(J9SHR_CACHELET_SUPPORT)
+static VMINLINE BOOLEAN j9shr_Query_IsCacheFull(J9JavaVM *vm)  __attribute__ ((__unused__));
+static VMINLINE BOOLEAN j9shr_Query_IsAddressInCache(J9JavaVM *vm, void *address, UDATA length)  __attribute__ ((__unused__));
+#endif /* !J9SHR_CACHELET_SUPPORT */
+static VMINLINE void j9shr_Query_PopulatePreinitConfigDefaults(J9JavaVM *vm, J9SharedClassPreinitConfig *updatedWithDefaults)  __attribute__ ((__unused__));
+#endif /* !J9VM_SHRTEST */
+static VMINLINE BOOLEAN j9shr_Query_IsAddressInReadWriteCache(J9JavaVM *vm, void *address, UDATA length) __attribute__ ((__unused__));
```

Comment:
This one should also be in !defined(J9VM_SHRTEST) && !defined(J9SHR_CACHELET_SUPPORT), it's in the same block as j9shr_Query_IsAddressInCache() below.

Reply:
Thanks for catching that. I've updated the PR